lib/spack/spack/cmd/list.py

```diff
@@ -125,6 +125,38 @@ def rows_for_ncols(elts, ncols):
             row.append(elts[i] if i < len(elts) else None)
         yield row
 
+def get_latest_version(version_sequence):
```

Comment:
This should probably use the same selection logic as `spack install <package>` instead of this crude selection.